src/components/CohortBuilder/Results.js

```diff
@@ -24,8 +24,26 @@ import { createFileRepoLink } from './util';
 import { injectState } from 'freactal';
 import saveSet from '@arranger/components/dist/utils/saveSet';
 import graphql from 'services/arranger';
-import { get } from 'lodash';
+import { get, isObject } from 'lodash';
 import { Link } from 'react-router-dom';
+import { trackUserInteraction, TRACKING_EVENTS } from 'services/analyticsTracking';
+
+const {
+  categories: {
+    cohortBuilder: {
+      views,
+    },
+  },
+  actions,
+} = TRACKING_EVENTS;
+
+const trackResultsAction = ({ category, action, label }) => {
+  trackUserInteraction({
+    category: `${views._cohortBuilderView}: ${category}`,
```

Comment:
When switching between Summary View and Table View the category ends up being redundant `Cohort Builder: View: Cohort Builder: View: Table`

Should just be `Cohort Builder: View: Table` maybe `Cohort Builder: Results View: Table`

Reply:
Added a `subcategory` param as follows to address this:
```javascript
{ category: category || `${views._cohortBuilderView}: ${subcategory}` }
```